.travis.yml

```diff
@@ -1,23 +1,69 @@
-sudo: required
+# File managed by pluginsync
+sudo: false
 language: go
 go:
-- 1.5.4
-- 1.6.2
+- 1.6.3
+- 1.7.1
 env:
   global:
-    - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/intelsdi-x/snap-plugin-collector-haproxy
+  - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
+  - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
   matrix:
-    - TEST=unit
+  - TEST_TYPE=small
+  - TEST_TYPE=build
```

Comment:
Should this matrix also include TEST_TYPEs legacy and medium? 
Currently there is no tests results for medium and legacy tests on Travis for this pull request and for the TEST_TYPE=build tests run only for go1.7.1 - I have doubt about that. Could you verify that?

![image](https://cloud.githubusercontent.com/assets/11335874/18686715/32449d48-7f7d-11e6-8554-5dd862977923.png)